src/background.js

```diff
@@ -106,6 +139,9 @@ const messageOps = {
     Metrics.updateSnoozedTab(message);
     return messageOps.cancel(message.old).then(() => messageOps.schedule(message.updated));
   },
+  setconfirm: message => {
+    browser.storage.local.set({dontShow:message.dontShow})
```

Comment:
Yeah, I couldn't figure out why that wasn't fixed on my following two pushes.  Turns out you need to `git add` before your `git commit`.  And on that note, I think I should stop writing code for the day. 😉